packages/react/src/Delay.spec.tsx

```diff
@@ -16,10 +16,11 @@ afterEach(() => {
 
 describe('<Delay/>', () => {
   it('should render the children after the delay', async () => {
-    render(<Delay ms={ms('0.1s')}>{TEXT}</Delay>)
-
+    render(<Delay ms={ms('0.5s')}>{TEXT}</Delay>)
     expect(screen.queryByText(TEXT)).not.toBeInTheDocument()
-    await waitFor(() => expect(screen.queryByText(TEXT)).toBeInTheDocument())
+    await vi.advanceTimersByTimeAsync(ms('0.5s'))
+    await vi.advanceTimersByTimeAsync(0)
```

Comment:
Should I call `advanceTimersBy TimeAsync(0)` here to get strict type checking? 😲

Reply:
If I remove `await vi.advanceTimersByTimeAsync(0)`, React renderer won't make new UI to test it
I don't really like this part either, but I think this is the right way to do it. I think it's because React uses timers internally.

Reply:
@manudeli I understand. Thank you for your hard work.